cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -538,37 +542,64 @@ class TemporalTest extends DocumentingTest {
-            || `duration.nanosecondsOfSecond` | Seconds | The total number of _nanoseconds_ in the group that do not make a whole _second_ | Integer
+            || `duration.monthsOfQuarter` | Months | The number of _months_ in the group that do not make a whole _quarter_ | Integer
+            || `duration.daysOfWeek` | Days | The number of _days_ in the group that do not make a whole _week_ | Integer
+            || `duration.minutesOfHour` | Seconds | The number of _minutes_ in the group that do not make a whole _hour_ | Integer
+            || `duration.secondsOfMinute` | Seconds | The number of _seconds_ in the group that do not make a whole _minute_ | Integer
+            || `duration.millisecondsOfSecond` | Seconds | The number of _milliseconds_ in the group that do not make a whole _second_ | Integer
+            || `duration.microsecondsOfSecond` | Seconds | The number of _microseconds_ in the group that do not make a whole _second_ | Integer
+            || `duration.nanosecondsOfSecond` | Seconds | The number of _nanoseconds_ in the group that do not make a whole _second_ | Integer
             ||===
             |
             |""")
-        p("The following query shows how to extract the components of a _Duration_ value:")
+        p("The following query shows how to extract the month based components of a _Duration_ value:")
         query(
-          """WITH duration({years: 1, months:4, days: 111, hours: 1, minutes: 1, seconds: 1, nanoseconds: 111111111}) AS d
-            |RETURN d.years, d.months, d.monthsOfYear, d.days, d.hours,
-            |   d.minutes, d.minutesOfHour, d.seconds, d.secondsOfMinute, d.milliseconds, d.millisecondsOfSecond, d.microseconds,
+          """WITH duration({years: 1, months:5, days: 111, minutes: 42}) AS d
+            |RETURN d.years, d.quarters, d.quartersOfYear, d.months, d.monthsOfYear, d.monthsOfQuarter""".stripMargin, ResultAssertions((r) => {
+            r.toList should equal(List(Map(
+              "d.years" -> 1,
+              "d.quarters" -> 5,
+              "d.quartersOfYear" -> 1,
+              "d.months" -> 17,
+              "d.monthsOfYear" -> 5,
+              "d.monthsOfQuarter" -> 2
+            )))
+
+          })) {
+          resultTable()
+        }
+        p("The following query shows how to extract the day based components of a _Duration_ value:")
+        query(
+          """WITH duration({months:5, days: 25, hours: 1}) AS d
+            |RETURN d.weeks, d.days, d.daysOfWeek""".stripMargin, ResultAssertions((r) => {
+            r.toList should equal(List(Map(
+              "d.weeks" -> 3,
+              "d.days" -> 25,
+              "d.daysOfWeek" -> 4
+            )))
+
+          })) {
+          resultTable()
+        }
+        p("The following query shows how to extract the second based components of a _Duration_ value:")
```

Comment:
The result table for this one is very squashed horizontally, and quite hard to read. Not sure if there is anything that can be done? Maybe split it into two queries?